build/link-wrapper.sh

```diff
@@ -32,6 +39,13 @@ case $1 in
 		    $CC $OPTIONS
 		fi
 		;;
+		freebsd)
+			CC=clang++
+			CFLAGS="-std=c++11 -stdlib=libc++ -msse4.2 -Wno-error=unused-command-line-argument -Wno-undefined-var-template -Wno-unknown-warning-option -Wno-varargs"
+			_LIBS="-lm -lc++ -lpthread -lexecinfo -ldevstat -lkvm"
```

Comment:
These should be in the makefile, and I think they are already, so you can probably remove them from here.

Reply:
Ok, thanks I will give that a try and see what happens. I need to look closer at what is happening with the build options. If something is going on there probably is a better solution than this to make sure that clang builds everything.

Reply:
This has been addressed here, looks like there is no way around other than checking to see if it's defined and if not then define it. As I was told "For Bourne shells (that includes Bash) variables need to be exported for 'child' processes to pick them up."

https://forums.freebsd.org/threads/makefiles-bash-and-the-cc-variable-with-clang.66719/

They have a "clean slate" development environment but that's what I like about it. Ok, this will get rewritten so that it checks for this at the top of the file.